provisioning/resources/control-plane/change_credentials.go

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
is go fmt run on those files? they seem to be space-indented instead of tab-indented?